generator.rb

```diff
@@ -0,0 +1,72 @@
+# establishing the run_generator method
+# getting all of the input needed
+# storing those pieces of input into variables
+# being able to print out those variables
+# iterating through individual characters in words to check which characters are vowels
+
+# wave 1:
+def is_vowel?(vowel)
+    vowels = %w{a e i o u}
+    if vowels.include?(vowel)
```

Comment:
Just a note:  Since `include?` returns true or false, you can condense your code with:

```ruby
return vowels.include?(vowel)
```